docs/paper/admin/reference/world-configuration.md

```diff
@@ -440,7 +440,9 @@ Recommended configuration for both `engine-mode` `1` and `2` can be found in the
 
 - **default**: `[Pos, Motion]`
 - **description**: A list of nbt paths that will be removed from the `EntityTag` tag on items which spawn entities.
-  The format of these strings follows the same format used to select nbt paths in vanilla commands. 
+  The format of these strings follows the same format used to select nbt paths in vanilla commands. This will only
+  apply to place actions which do not have a player involved or for players who do not have the `minecraft.nbt.place`
+  permission. If a player with that permission places an entity, this filter list will be ignored.
```

Comment:
mmh, I feel like this could be better worded. can't we just say, "Players with the blah permission ignore this filter"?